projects/js-packages/publicize-components/src/components/connection-icon/style.scss

```diff
@@ -11,6 +11,9 @@
 	text-overflow: ellipsis;
 	white-space: nowrap;
 
+	.jetpack-publicize-gutenberg-social-icon {
+		margin-right: 10px;
```

Comment:
I'm wondering if the size of the icon is wrong since its move to the package, as they were displaying well without the a profile image. The right margin is set on the icon at the top of the file, so if this is the default, perhaps we should have it there?

Reply:
Yeah, I can switch the two numbers around. The size isn't wrong per se, but the image + icon combination is wider than just the icon. I could get a bit fancy and make a grid structure for all items but that might be more trouble than it's worth right now.

Reply:
> the image + icon combination is wider than just the icon

Yes, that makes me wonder if we should just make the icon a few pixels bigger. I've got a vague memory of it being 16px square, and it's only 15px currently so maybe that would fix things.

Adjust the margin like this is fine too of course!

Reply:
I'll try making the icon bigger, it might look better if it's the same size as the profile picture. Even then, we'll need to have a different margin on the right to adjust for the smaller icon in the corner of a profile picture.

Reply:
There's something odd going on with the icons, they seem to have some inner padding hardcoded in the SVG files. I did fix the margin in [9b9dd43](https://github.com/Automattic/jetpack/pull/24736/commits/9b9dd4384a693f8b94028e04bcfc8d4b37bc8c02).